### DISCUSSION TITLE
Cowrie - server refused to allocate pty

### DISCUSSION THREAD

**Discussion by OliHebden** [telekom-security/tpotce#780] [Q&A]
Hi,

So I've customised cowrie with a custom userdb.txt, rebuilt the cowrie image and its up and running. I used "docker exec -it cowrie sh" to check the running container had the new userdb file and then attempted to connect via ssh to check the credentials.

They worked but I was met with the error "server refused to allocate pty" when I used the correct credentials. I've checked "lsof /dev/pts/*" in the running container and there isn't any zombie connections so I'm a bit confused as to why I'm getting this error?

Any advice would be helpful, thanks.

**Comment by t3chn0m4g3** [comment]
This seems a little more Cowrie related. You should check their [Slack ](https://www.cowrie.org/slack/)channel and [GitHub ](https://github.com/cowrie/cowrie)page.
